### ISSUE TITLE
Remove logging output

### ISSUE THREAD

**Issue by brentryan** [k9securityio/k9-cdk#23]
Currently, some logging output is printed out when running cdk synth / deploy in relation to k9-cdk package.

```
trustAccountIdentities: undefined
Omitting Allow root and DenyEveryoneElse statements
```

**Comment by skuenzli** [comment]
Yeah, I'll probably go ahead and remove those.  They are meant as `Notice:` type messages.

I looked (again) for guidance on logging within constructs in the CDK developer guide.  I could only find this issue saying 
https://github.com/awsdocs/aws-cdk-guide/issues/99 something had been documented, but folks can't seem to find the guidance.

**Comment by brentryan** [comment]
Ya, debugging CDK code could be better.

**Comment by coltenkrauter** [comment]
I'm a bit confused. If I add a `console.error('msg')`, should it log to the console during `cdk synth`? It is not working for me.

**Comment by skuenzli** [comment]
I'm not sure  if `console.error` is expected to work.  But last I checked the aws-cdk source code, I didn't see any logging at all.

The [datadog construct library](https://github.com/DataDog/datadog-cdk-constructs/blob/main/v2/package.json) brings in the loglevel package:
https://www.npmjs.com/package/loglevel

**Comment by skuenzli** [comment]
Fixed in `59b65df4`
